src/mugs.js

```diff
@@ -1464,6 +1464,22 @@ define([
                     return data.id && data.id.slice(data.id.lastIndexOf("/") + 1);
                 }
             },
+            labelItext: {
```

Comment:
nevermind. I see this is for choices only. I suppose this is a reasonable place to put it.